remote/gogs/helper.go

```diff
@@ -66,13 +66,29 @@ func buildFromPush(hook *PushHook) *model.Build {
 		hook.Repo.Url,
 		fixMalformedAvatar(hook.Sender.Avatar),
 	)
+
+	eventType := "Undefined"
+	message := "No message"
+
+	switch {
+	case hook.RefType == "tag":
+		eventType = model.EventTag
+		message = "Create Tag " + hook.Ref
+	case hook.RefType == "branch":
+		eventType = model.EventPush
+		message = "Create Branch " + hook.Ref
+	default:
```

Comment:
should a branch creation event trigger a build in drone? with GitHub we ignore these, because otherwise it would trigger two builds, since we get an event for branch creation AND for the push